backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/UpdateClusterUpgradeCommand.java

```diff
@@ -0,0 +1,79 @@
+package org.ovirt.engine.core.bll;
+
+import javax.inject.Inject;
+
+import org.ovirt.engine.core.bll.context.CommandContext;
+import org.ovirt.engine.core.common.AuditLogType;
+import org.ovirt.engine.core.common.action.ClusterUpgradeParameters;
+import org.ovirt.engine.core.common.businessentities.Cluster;
+import org.ovirt.engine.core.common.errors.EngineMessage;
+import org.ovirt.engine.core.compat.Guid;
+import org.ovirt.engine.core.dao.ClusterDao;
+import org.ovirt.engine.core.utils.transaction.TransactionSupport;
+
+@NonTransactiveCommandAttribute(forceCompensation = true)
+public class UpdateClusterUpgradeCommand<T extends ClusterUpgradeParameters> extends ClusterCommandBase<T> {
+
+    @Inject
+    private ClusterDao clusterDao;
+
+    private Cluster cluster;
+
+    @Override
+    protected void init() {
+        cluster = clusterDao.get(getCluster().getId());
+    }
+
+    public UpdateClusterUpgradeCommand(T parameters, CommandContext commandContext) {
+        super(parameters, commandContext);
+    }
+
+    protected UpdateClusterUpgradeCommand(Guid commandId) {
+        super(commandId);
+    }
+
+    @Override
+    protected void executeCommand() {
+        TransactionSupport.executeInNewTransaction(() -> {
+            int progress = getParameters().getUpgradeProgress();
+            boolean updated = clusterDao.setUpgradeProgress(cluster.getId(), progress);
+
+            if (updated) {
+                log.info("Set upgrade progress to {} on cluster '{}'.", progress, cluster.getId());
```

Comment:
Is this message really needs to be at INFO level? It seems more to me like a DEBUG stuff

Reply:
Ack